staging_docs/admin/reference/settings.md

```diff
@@ -0,0 +1,71 @@
+# Settings
+
+`pulp_ansible` provides a few settings to control various features. These settings are settable
+with dynaconf in various as a regular Pulp setting. See the [pulpcore Setting](https://docs.pulpproject.org/en/3.0/nightly/installation/configuration.html#configuration).
```

Comment:
I do not like this line break policy. It is not helpful and presents a sizeable entry barrier for contributors. I cannot even check it in a meaningful way in a PR.

Reply:
I don't understand what you are saying. Are you talking about the number of characters allowed per line? I thought that was just a part of the lint rules.

Reply:
I'm talking about the fact that linebreaks keep happening in the middle of a sentence (not at syntactical breakpoints). And then when something like a URL comes into play, we simply abandon the rule to break at 100 c.

Reply:
While I think this is worth a discussion - it's not the goal of *this* PR, which is "get the docs into the new docs-stack". We'll have plenty of cleanup to do after that :)